README.md

```diff
@@ -47,6 +47,14 @@ docker:
     - "/path/to/another-image-name.tar"
 ```
 
+### Run a Command
+device-init can execute a list of commands on boot. e.g.:
+```yaml
+runcmd:
+  - "apt-get install package-name"
```

Comment:
interesting - my assumption was that `ls -la | grep test` will not work because of the pipe symbol. I tried some more examples and it is quite hard to produce yaml syntax errors. But after trying hard enough it is possible e.g.
```
runcmd:
 - echo hello: > /tmp/test-without-quotes.txt
```
this yaml has syntax error because of colon followed by space (you can also check on http://www.yamllint.com/), while 
```
runcmd:
 - "echo hello: > /tmp/test-with-quotes.txt"
```
works.

so, at least I will change the test to make sure it requires the quotes and try to make the tests more descriptive (e.g. adding comments).